dataplug-instagram/app/com/hubofallthings/dataplugInstagram/apiInterfaces/InstagramFeedInterface.scala

```diff
@@ -146,6 +152,24 @@ class InstagramFeedInterface @Inject() (
     }
   }
 
+  private def transformData(rawData: JsValue): JsResult[JsObject] = {
+    val transformation = (__ \ "data").json.update(
+      __.read[JsArray].map { feedData =>
+        {
+          val updatedFeedData = feedData.value.map { item =>
+            val unixTimeStamp = DateTime.parse((item \ "timestamp").asOpt[String].getOrElse("")).getMillis / 1000
+            item.as[JsObject] ++ JsObject(Map(
+              "api_version" -> JsString("v2"),
+              "created_time" -> JsString(unixTimeStamp.toString)))
```

Comment:
Both fields should start with a `ds` prefix to clearly indicate organisation adding these fields into the record. Prefix should be configurable through the environment variables with a default for `ds`.

Reply:
Sure no problem. Why should it be configurable? What's the use case?